lib/ex_pix_brcode/brcode/models/brcode.ex

```diff
@@ -75,11 +72,13 @@ defmodule ExPixBRCode.Models.BRCode do
       required: true
     )
     |> validate_required(@required)
-    |> validate_inclusion(:payload_format_indicator, ["01"])
-    |> validate_inclusion(:point_of_initiation_method, ["12"])
+    |> validate_inclusion(:payload_format_indicator, ~w(01))
+    |> validate_inclusion(:point_of_initiation_method, ~w(11 12))
     |> validate_format(:merchant_category_code, ~r/^[0-9]{4}$/)
-    |> validate_inclusion(:transaction_currency, ["986"])
-    |> validate_inclusion(:country_code, ["BR"])
+    |> validate_inclusion(:transaction_currency, ~w(986))
+    |> validate_length(:transaction_amount, max: 13)
+    |> validate_format(:transaction_amount, ~r/(^0$)|(^[0-9]+\.[0-9]*$)/)
```

Comment:
```suggestion
    |> validate_format(:transaction_amount, ~r/(^[0-9]+(\.[0-9]*)?$)/)
```

Your regex does not allow "00" or "01". Is this intentional? If not, My suggestion allows for these cases as well as "00." and "00.0"

Reply:
It is intentional. According to the specification of BACEN and the EMV-QRCPS we must allow:

- 0
- 01.
- 100.00

Reply:
Is "01" allowed by the spec?

Reply:
Maybe there should be a link to the spec as a comment in the code?

Reply:
The link is, unfortunatelly, very dynamic. I can "mention it" for sure.

Only "0" has special semantics. All other must have one ".".